app.js

```diff
@@ -0,0 +1,16 @@
+const express = require("express")
+const app = express()
+const { getTopics } = require("./controllers/controller")
+
+
+app.get("/api/topics", getTopics)
+
+app.use((err, req, res, next) => {
+    if (err.status && err.message) {
+        res.status(err.status).send({ message: err.message })
+    } else {
+        res.status(500).send({ message: 'internal server error' })
+    }
+})
```

Comment:
You should not need to have custom error handling at this stage. Having a 500 internal server error is fine, just so you have something to fall back on but that is all this error handler should have.